sncosmo/fitting.py

```diff
@@ -662,6 +654,11 @@ def fit_lc(data=None, model=None, vparam_names=[], bounds=None, spectra=None,
             _print_iminuit_params(vparam_names, kwargs)
             print()
 
+        # Set up keyword arguments to Migrad.minuit
+        migrad_kwargs = {"ncall": maxcall}
+        if version_parse(iminuit.__version__) >= version_parse("1.5.2"):
+            migrad_kwargs["iterate"] = 1
```

Comment:
Impressive investigation in https://github.com/scikit-hep/iminuit/issues/499! From that issue it sounds like iminuit 1.5.x has an uncertain status. I'm wondering if this is the "right" fix for all future iminuit versions >= 1.5.2?

Reply:
Good point, the better solution is probably to avoid the 1.5.2 release altogether. I'll withdraw this commit from the PR.